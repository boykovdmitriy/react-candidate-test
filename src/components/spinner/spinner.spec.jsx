import React from 'react';
import {Spinner} from './index';
import {shallow} from 'enzyme/build/index';

describe('Spinner', () => {
  it('render Spinner', () => {
    const tree = shallow(<Spinner/>);
    expect(tree).toMatchSnapshot();
  });
});
