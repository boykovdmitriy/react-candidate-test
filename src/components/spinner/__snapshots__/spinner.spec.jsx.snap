// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner render Spinner 1`] = `
<section
  className="spinner"
>
  ...Loading
</section>
`;
